Zend/Zend.m4

```diff
@@ -596,3 +596,21 @@ dnl This is the most probable fallback so we assume yes in case of cross compile
 if test "$ac_cv_huge_val_nan" = "yes"; then
   AC_DEFINE([HAVE_HUGE_VAL_NAN], 1, [whether HUGE_VAL + -HUGEVAL == NAN])
 fi
+
+dnl
+dnl Check whether __cpuid_count is available
+dnl
+AC_CACHE_CHECK(whether __cpuid_count is available, ac_cv_cpuid_count_available, [
+AC_LINK_IFELSE([AC_LANG_PROGRAM([[
+  #include <cpuid.h>
+]], [[
+  unsigned eax, ebx, ecx, edx;
+  __cpuid_count(0, 0, eax, ebx, ecx, edx);
+]])], [
```

Comment:
The double `[` and `]` might not be needed for the program body. See https://www.gnu.org/software/autoconf/manual/autoconf-2.64/html_node/Quotation-Rule-Of-Thumb.html, but otherwise is fine, too.

Reply:
Hmm, the examples in the [AC_LANG_PROGRAM docs](https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Generating-Sources.html) as well as the already existing occurrences of AC_LANG_PROGRAM in Zend.m4 use double quotes for the first two arguments.